hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -199,116 +245,87 @@ def check_all_ports_firewall_rules(reason=''):
 def world_writable_file(reason=''):
     '''
     Ensure no world writable files exist
     '''
     result = _execute_shell_command('df --local -P | awk {\'if (NR!=1) print $6\'} | xargs -I \'{}\' find \'{}\' -xdev -type f -perm -0002')
-    if result == '':
-      return True
-    return result
+    return True if result == '' else result
+
 
 def system_account_non_login(reason=''):
     '''
     Ensure system accounts are non-login
     '''
     result = _execute_shell_command('egrep -v "^\+" /etc/passwd | awk -F: \'($1!="root" && $1!="sync" && $1!="shutdown" && $1!="halt" && $3<500 && $7!="/sbin/nologin" && $7!="/bin/false") {print}\'')
-    if result == '':
-      return True
-    return result
+    return True if result == '' else result
+
 
 def sticky_bit_on_world_writable_dirs(reason=''):
     '''
     Ensure sticky bit is set on all world-writable directories
     '''
     result = _execute_shell_command('df --local -P | awk {\'if (NR!=1) print $6\'} | xargs -I \'{}\' find \'{}\' -xdev -type d \( -perm -0002 -a ! -perm -1000 \) 2>/dev/null')
-    if result == '':
-      return True
-    return "There are failures"
+    return True if result == '' else "There are failures"
+
 
 def default_group_for_root(reason=''):
     '''
     Ensure default group for the root account is GID 0
     '''
     result = _execute_shell_command('grep "^root:" /etc/passwd | cut -f4 -d:')
     result = result.strip()
-    if result == '0':
-      return True
-    return False
+    return True if result == '0' else False
+
 
 def root_is_only_uid_0_account(reason=''):
     '''
     Ensure root is the only UID 0 account
     '''
     result = _execute_shell_command('cat /etc/passwd | awk -F: \'($3 == 0) { print $1 }\'')
-    if result.strip() == 'root':
-      return True
-    return result
+    return True if result.strip() == 'root' else result
 
-def test_success():
-    '''
-    Automatically returns success
-    '''
-    return True
-
-
-def test_failure():
-    '''
-    Automatically returns failure, no reason
-    '''
-    return False
-
-
-def test_failure_reason(reason):
-    '''
-    Automatically returns failure, with a reason (first arg)
-    '''
-    return reason
 
 def test_mount_attrs(mount_name,attribute,check_type='hard'):
     '''
     Ensure that a given directory is mounted with appropriate attributes
     If check_type is soft, then in absence of volume, True will be returned
     If check_type is hard, then in absence of volume, False will be returned
     '''
-    #check that the path exists on system
+    # check that the path exists on system
     command = 'test -e ' + mount_name + ' ; echo $?'
     output = _execute_shell_command( command)
     if output.strip() == '1':
         return True if check_type == "soft" else (mount_name + " folder does not exist")
 
-    #if the path exits, proceed with following code
+    # if the path exits, proceed with following code
     output = _execute_shell_command('mount | grep ' + mount_name)
```

Comment:
Any time when we pass variables to a function that executes them, it's susceptible to command injection. It's best to avoid these situations.

Reply:
Got it.